source/Calamari/Commands/DeployPackageCommand.cs

```diff
@@ -53,8 +53,8 @@ public override int Execute(string[] commandLineArguments)
             var fileSystem = new CalamariPhysicalFileSystem();
             var scriptEngine = new ScriptEngineSelector();
             var replacer = new ConfigurationVariablesReplacer();
+            var configurationTransformer = new ConfigurationTransformer(variables.GetFlag(SpecialVariables.Package.IgnoreConfigTransformationErrors));
             var substituter = new FileSubstituter();
-            var configurationTransformer = new ConfigurationTransformer();
```

Comment:
Oh, wasn't intentional - I was originally working off an older version that didn't have `variables` defined. I moved it below that then the merge changed it all around. I'll fix :)